contracts/payloads/IGP95/PayloadIGP95.sol

```diff
@@ -0,0 +1,973 @@
+
+                VAULT_FACTORY.setVaultAuth(
+                    PAXG_GHO_VAULT,
+                    TEAM_MULTISIG,
+                    false
+                );
+            }
+        }
+
+        {
+            // [TYPE 2] PAXG-XAUT<>USDC | smart collateral & normal debt
+            address PAXG_XAUT__USDC_VAULT = getVaultAddress(122);
+
+            {
+                VaultConfig memory VAULT_PAXG_XAUT__USDC = VaultConfig({
+                    vault: PAXG_XAUT__USDC_VAULT,
+                    vaultType: VAULT_TYPE.TYPE_2,
+                    supplyToken: address(0),
+                    borrowToken: USDC_ADDRESS,
+                    baseWithdrawalLimitInUSD: 0,
+                    baseBorrowLimitInUSD: 1_000_000, // $1M
+                    maxBorrowLimitInUSD: 2_000_000 // $2M
+                });
+
+                setVaultLimits(VAULT_PAXG_XAUT__USDC); // TYPE_2 => 122
+
+                VAULT_FACTORY.setVaultAuth(
+                    PAXG_XAUT__USDC_VAULT,
+                    TEAM_MULTISIG,
+                    false
+                );
+            }
+        }
+
+        {
+            // [TYPE 2] PAXG-XAUT<>USDT | smart collateral & normal debt
+            address PAXG_XAUT__USDT_VAULT = getVaultAddress(123);
+
+            {
+                VaultConfig memory VAULT_PAXG_XAUT__USDT = VaultConfig({
+                    vault: PAXG_XAUT__USDT_VAULT,
+                    vaultType: VAULT_TYPE.TYPE_2,
+                    supplyToken: address(0),
+                    borrowToken: USDT_ADDRESS,
+                    baseWithdrawalLimitInUSD: 0,
+                    baseBorrowLimitInUSD: 1_000_000, // $1M
+                    maxBorrowLimitInUSD: 2_000_000 // $2M
+                });
+
+                setVaultLimits(VAULT_PAXG_XAUT__USDT); // TYPE_2 => 123
+
+                VAULT_FACTORY.setVaultAuth(
+                    PAXG_XAUT__USDT_VAULT,
+                    TEAM_MULTISIG,
+                    false
+                );
+            }
+        }
+
+        {
+            // [TYPE 2] PAXG-XAUT<>GHO | smart collateral & normal debt
+            address PAXG_XAUT__GHO_VAULT = getVaultAddress(124);
+
+            {
+                VaultConfig memory VAULT_PAXG_XAUT__GHO = VaultConfig({
+                    vault: PAXG_XAUT__GHO_VAULT,
+                    vaultType: VAULT_TYPE.TYPE_2,
+                    supplyToken: address(0),
+                    borrowToken: GHO_ADDRESS,
+                    baseWithdrawalLimitInUSD: 0,
+                    baseBorrowLimitInUSD: 1_000_000, // $1M
+                    maxBorrowLimitInUSD: 2_000_000 // $2M
+                });
+
+                setVaultLimits(VAULT_PAXG_XAUT__GHO); // TYPE_2 => 124
+
+                VAULT_FACTORY.setVaultAuth(
+                    PAXG_XAUT__GHO_VAULT,
+                    TEAM_MULTISIG,
+                    false
+                );
+            }
+        }
+    }
+
+    // @notice Action 10: Update Borrow Limits for sUSDe/USDC
```

Comment:
@xma8lu @bergben Are these dust limits?

Reply:
no this is on a retired v1 vault, we only need to increase this so we can balance out the outstanding magnifier fees, vault borrows from LL and transfers to reserve contract which is currently not possible because of the limit

![image](https://github.com/user-attachments/assets/f6cffbc6-d6d8-43e2-84d4-d9dfa0f9f8fd)

Reply:
hey! we needed to do this in order to properly wind down the vault as stated by Ben
![Uploading Screenshot 2025-05-10 at 11.39.51 PM.png…]()